draft-ietf-rats-architecture.md

```diff
@@ -813,6 +813,9 @@ component or user on the device.
 
 ## Attestation Results
 
+Attestation Results are the input used by the Relying Party to decide whether it will trust a particular device, allow access to some content or data, allow a financial transaction or other.
+Attestation Results may be a Boolean simply indicating compliance or non-compliance, or a rich set of Claims about the device, against which the Relying Party applies its Appraisal Policy for Attestation Results.
```

Comment:
```suggestion
Attestation Results may be a Boolean simply indicating compliance or non-compliance, or a rich set of Claims about the Attester, against which the Relying Party applies its Appraisal Policy for Attestation Results.
```
"device" -> "Attester".   Also I think this sentence is redundant with the next sentence so rather than causing duplication, I think it should be merged with the next sentence.